### ISSUE TITLE
Body background color does not change when using "dark" theme

### ISSUE THREAD

**Issue by SaadBazaz** [primer/react#2370]
### Description

The background color of the document's body should automatically update if I select the "dark", "night" or similar themes.

I don't know if this is **expected** behaviour. If it isn't, I'd certainly want it to be.

This is what I see in normal mode:
<img width="1264" alt="Screenshot 2022-09-24 at 10 40 22 AM" src="https://user-images.githubusercontent.com/51885228/192081946-6869451b-ba14-40f4-8e5c-ebe43645e476.png">

But this is what I see if I simply use ```colorMode={"dark"}```:
<img width="1266" alt="Screenshot 2022-09-24 at 10 41 49 AM" src="https://user-images.githubusercontent.com/51885228/192081982-adad78ec-1f15-4edc-9b50-238302276812.png">

**However**, if I manually set ```backgroundColor: "black"``` in index.css, it looks good:
<img width="1262" alt="Screenshot 2022-09-24 at 10 42 37 AM" src="https://user-images.githubusercontent.com/51885228/192082002-879eee49-07fb-4f1d-b4d6-4ccd5e52f475.png">

### Steps to reproduce

1. Create a fresh React app through ```create-react-app```
2. Install ```primer```
3. Add ```ThemeProvider``` to App.js
4. Use the "dark" theme

### Version

v35.10.0

### Browser

Chrome, Safari

**Comment by devephy** [comment]
Did you find the help you needed?

**Comment by SaadBazaz** [comment]
Nope. Not yet.

**Comment by siddharthkp** [comment]
Hi! 

Yep, this is intentional. ThemeProvider only affects the contents inside of it. 

You can either wrap the entire page in a ThemeProvider and use `canvas.default` to set the background ([here's an example](https://codesandbox.io/s/dreamy-sun-7jmw63?file=/src/Playground.js:560-592)) or you can set styles on the body with [styled-components#createglobalstyle](https://styled-components.com/docs/api#createglobalstyle) and [themeGet from primer/react](https://primer.style/react/theming#themeget)

**Comment by siddharthkp** [comment]
I hope that answers your question! 

Feel free to reply if not and I'll open the issue again

**Comment by SaadBazaz** [comment]
@siddharthkp got it! Makes sense anyway. I'll work on what you suggested. 

About the screenshots; it's something I'm building which might be of interest to GH.

Let's talk, maybe?

**Comment by siddharthkp** [comment]
Yeah, feel free to [DM me](https://twitter.com/siddharthkp) :)

**Comment by Zwyx** [comment]
Hey guys,

I was also quite surprised with this behaviour, and I spent quite some time reading the doc again and again trying understand what I was doing wrong for my background to not follow the dark theme.

Thanks to your messages here, I wrapped everything inside a `<Box bg="canvas.default" minHeight="100vh">`.

I believe an example like this really deserves a spot in the documentation at https://primer.style/react/theming#color-modes-and-color-schemes

I'm keen to create a PR if it's something you would agree with?
